src/core/hyprlock.cpp

```diff
@@ -530,6 +530,24 @@ void CHyprlock::run() {
     Debug::log(LOG, "Reached the end, exiting");
 }
 
+void CHyprlock::unlock() {
```

Comment:
naming gets confusing here, `unlockChecked` and `unlockUnchecked`, maybe? Or something?

Reply:
Agree. How about renaming `lockSession` and `unlockSession` to smth like `acquireSessionLock` and `releaseSessionLock`?

Reply:
sure